cernopendata/templates/cernopendata_theme/header.html

```diff
@@ -2,7 +2,7 @@
 {%- block navbar %}
 <nav class="navbar-custom">
   <a class="navbar-brand" href="{{ url_for('cernopendata_pages.index') }}">
-    <img src="{{ url_for('static', filename='assets/img/logo_blue.png') }}" width="115" alt="">
+    <img src="{{ url_for('static', filename='assets/img/logo_blue.png') }}" width="115" alt="home">
```

Comment:
same comment if it is needed to fill-in the alt or leave it empty...

Reply:
```suggestion
    <img src="{{ url_for('static', filename='assets/img/logo_blue.png') }}" width="115" alt="">
```